src/DataGrid.tsx

```diff
@@ -548,16 +548,27 @@ function DataGrid<R, SR, K extends Key>(
     if (!isCellEvent && !isRowEvent) return;
 
     const { keyCode } = event;
+    // event.key may differ by keyboard input language, so we use event.keyCode instead
+    // event.nativeEvent.code cannot be used either as it would break copy/paste for the DVORAK layout
+    const cKey = 67;
+    const vKey = 86;
+
+    const selection = window.getSelection();
+    // pressing ctrl+c on highlighted text should copy text instead of starting editing
+    if (
+      isCtrlKeyHeldDown(event) &&
+      keyCode === cKey &&
+      selection !== null &&
+      selection.toString() !== ''
```

Comment:
🤔 
https://developer.mozilla.org/en-US/docs/Web/API/Selection/isCollapsed
```suggestion
      window.getSelection()?.isCollapsed === false
```